src/layouts/myrift.pug

```diff
@@ -0,0 +1,12 @@
+extends ../base.pug
+
+block prepend vars
+  -var pageClass = "myrift"
```

Comment:
You shouldn't a layout for Hx and RIFT: set this variable as part of the case study isn't.